library/core/src/slice/mod.rs

```diff
@@ -3293,8 +3293,9 @@ mod private_slice_index {
     on(T = "str", label = "string indices are ranges of `usize`",),
     on(
         all(any(T = "str", T = "&str", T = "std::string::String"), _Self = "{integer}"),
-        note = "you can use `.chars().nth()` or `.bytes().nth()`
-see chapter in The Book <https://doc.rust-lang.org/book/ch08-02-strings.html#indexing-into-strings>"
+        note = "you can use `.chars().nth()` or `.bytes().nth()`\n\
+                For more information, see chapter 8 in The Book: \
```

Comment:
Nit: this is changing it to visual indent (following the position of the `"`"), but it should be block-indented instead, per https://github.com/rust-dev-tools/fmt-rfcs/blob/master/guide/principles.md#overarching-guidelines

So something like 
```
    on(
        all(any(T = "str", T = "&str", T = "std::string::String"), _Self = "{integer}"),
        note = "you can use `.chars().nth()` or `.bytes().nth()`\n\
            For more information, see chapter 8 in The Book: \
```
(Note how the `all` is indented 4, not 3 as it would be if it were following `on(`)

Reply:
That case is more like:
```
// Block indent
a_function_call(foo,
    bar,
);
```